api/nakadi-event-bus-api.yaml

```diff
@@ -1567,13 +1652,14 @@ definitions:
           future there could be others.
       schema:
         type: string
-        $ref: '#/definitions/EventTypeSchema'
         description: |
           The schema as string in the syntax defined in the field type. Failure to respect the
           syntax will fail any operation on an EventType.
 
-          To have a generic, undefined schema it is possible to define the schema as `"schema":
-          "{\"additionalProperties\": true}"`.
+          When defining a schema, every property should be explicitly defined with its type. It means that,
+          additionalProperties true is not accepted.
```

Comment:
@v-stepanov they will be marked with `compatibility_mode: deprecated` and will keep working as usual. Upgrading them would be possible since that would mean that events from the same event type could have incompatible fields.

Event if additionalProperties true is not explicitly defined, it's the default semantics for validation. So I don't believe we can solve this. New event types with proper schema and compatibility modes must be created for every event type.

But that just sounds terrible. So, we might open an exception and allow an "upgrade" in the compatibility mode, given everybody involved is aware that old (not versioned) event types are going to be incompatible with new (1.0 on) events.